app/views/toolbar.js

```diff
@@ -392,7 +392,7 @@ module.exports = Backbone.View.extend({
         // Finally, clear the queue object
         this.queue = undefined;
       } else {
-        var src = '{{site.baseurl}}/' + $('input[name="url"]').val();
+        var src = '{{site.baseurl}}' + $('input[name="url"]').val();
```

Comment:
why not something like `path.join` here? not sure if that' possible in the browser. maybe create a URI/URL and use that to join them? ideally it would be able to handle both cases.